components/service-operator/apis/storage/v1beta1/s3_types_test.go

```diff
@@ -125,8 +125,58 @@ var _ = Describe("S3Bucket", func() {
 				Expect(doc.Statement).To(HaveLen(1))
 				statement := doc.Statement[0]
 				Expect(statement.Effect).To(Equal("Allow"))
-				Expect(statement.Action).To(ContainElement("s3:ListObjects"))
-				Expect(statement.Action).To(ContainElement("s3:*Object"))
+				Expect(statement.Action).To(ConsistOf(
+					"s3:GetObjectVersionTagging",
+					"s3:ReplicateObject",
+					"s3:GetObjectAcl",
+					"s3:GetBucketObjectLockConfiguration",
+					"s3:GetObjectVersionAcl",
+					"s3:HeadBucket",
+					"s3:DeleteObject",
+					"s3:GetBucketPolicyStatus",
+					"s3:GetObjectRetention",
+					"s3:GetBucketWebsite",
+					"s3:ListJobs",
+					"s3:PutObjectLegalHold",
+					"s3:GetObjectLegalHold",
+					"s3:GetBucketNotification",
+					"s3:GetReplicationConfiguration",
+					"s3:ListMultipartUploadParts",
+					"s3:PutObject",
+					"s3:GetObject",
+					"s3:DescribeJob",
```

Comment:
Do we need this?

Reply:
I was only fussy about the "write" and "permisison setting" policies ... all read stuff is ok